app/main/main.js

```diff
@@ -10,7 +10,8 @@ function createWindow () {
     webPreferences: {
       preload: path.join(__dirname, 'preload.js'),
       webviewTag: true
-    }
+    },
+    /* icon: path.join(__dirname, "../icons/freecat-2.icns") */
```

Comment:
Nice, could you add an icon for the rest of us?

Reply:
I'm working on it, doesn't seem to work though...

Reply:
> 

Just use `<link rel="shortcut icon">`

Reply:
Oh, okay, that works, I'll try it!